Develop

martimalek opened the pull request:
Precourse

lexbarba commented:
Nice game Marti!
I have moved two variables (usedNums & userArray) at the top of the code so that when the functions that need them, the variables are declared.

I have also moved the function generateArray() at the top of the code, to be the first one to be executed. As it was executed later, the bingo did not find any match as the array userArray was not declared yet.

And that is it! Nice work!

martimalek commented:
> Nice game Marti!
> I have moved two variables (usedNums & userArray) at the top of the code so that when the functions that need them, the variables are declared.
> 
> I have also moved the function generateArray() at the top of the code, to be the first one to be executed. As it was executed later, the bingo did not find any match as the array userArray was not declared yet.
> 
> And that is it! Nice work!

Done! THX!!!
